app/javascript/src/Common/threescale.ts

```diff
@@ -1,4 +1,7 @@
+import $ from 'jquery'
+
 window.ThreeScale = {
+  activeAjaxRequests: () => $.active + window.$.active,
```

Comment:
so jquery 1 is still here but we catch both jquery 1 and 2 active requests, correct?

Reply:
1 an 3 exactly, but yes!